src/MobSpawner.cpp

```diff
@@ -340,6 +290,109 @@ bool cMobSpawner::CanSpawnHere(cChunk * a_Chunk, int a_RelX, int a_RelY, int a_R
 
 
 
+std::set<eMonsterType> cMobSpawner::GetAllowedMobTypes(EMCSBiome a_Biome)
+{
+	std::set<eMonsterType> ListOfSpawnables;
+	// Check biomes first to get a list of animals
+	if (a_Biome == biNether)
+	{
+		ListOfSpawnables.insert(mtGhast);
+		ListOfSpawnables.insert(mtMagmaCube);
+		ListOfSpawnables.insert(mtZombiePigman);
+		ListOfSpawnables.insert(mtEnderman);
+	}
+	else if (a_Biome == biEnd)
```

Comment:
Is there a reason you've moved away from `switch` to `if`-`else` chains?

Reply:
Looks like I misread your last review :/